### PR TITLE
Tests for Modal component

### PR DESCRIPTION
**Pull request by OSPFNeighbour**
tenor 1

### CODE REVIEW

File: client/src/components/Modal/__tests__/Modal.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+const noop = () => {};
+
+const child = (
+  <Text>test</Text>
+);
+
+test('renders correctly', () => {
+  const tree1 = renderer.create(
+    <Modal
```
Comment (sdunster) [inline client/src/components/Modal/__tests__/Modal.js]: lol wut did you forget how to nest components?
```
<Modal title="test" closeModal={noop} visible>
  {child}
</Modal>
```

File: client/src/components/Modal/__tests__/NumberInputModal.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import renderer from 'react-test-renderer';
+
+import NumberInputModal from '../NumberInputModal';
+
+const noop = () => {};
+
+test('renders correctly', () => {
+  const tree1 = renderer.create(
+    <NumberInputModal
+      title="test"
```
Comment (sdunster) [inline client/src/components/Modal/__tests__/NumberInputModal.js]: if you put different text in the attributes then we can more easily detect someone mixing prop names up. ie if you set both `title="test"` and `placeHolder="test"` and someone puts `{placeHolder}` where there is supposed to be `{title}` then the snapshot test wouldn't catch that bug. If the attributes are all different there would be a change that the snapshot test would catch. As such I recommend doing what I did for the others - for string props put the name of the prop as the test value.

File: client/src/components/Modal/__tests__/TextInputModal.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+import renderer from 'react-test-renderer';
+
+import TextInputModal from '../TextInputModal';
+
+const noop = () => {};
+
+test('renders correctly', () => {
+  const tree1 = renderer.create(
+    <TextInputModal
+      title="test"
```
Comment (sdunster) [inline client/src/components/Modal/__tests__/TextInputModal.js]: `title="title"`

File: client/src/components/Modal/__tests__/Modal.js (hunk 1 above)
Comment (OSPFNeighbour) [reply client/src/components/Modal/__tests__/Modal.js]: ooh is that what it meant. duh.

**Pull request merged**